pbspro.spec

```diff
@@ -194,7 +196,9 @@ HPC clusters, clouds and supercomputers.
 This package is intended for a client host and provides
 the PBS Professional user commands.
 
-%if 0%{!?opensuse_bs}
+%if 0%{?opensuse_bs}
```

Comment:
Is this change related to the ticket being addressed in this pull request? If not, I suggest we create another ticket to address this.